### PR TITLE
Clean up licensing information

### LINKED ISSUE #77: Our logo icon is currently under a non-free license
The Non-Commercial aspect of this license is problematic (for several reasons, one of which is the packaging of the program in free software operating system distributions). We should consider changing the licensing terms to a libre license. I suggest adopting the Creative Commons By-SA license for these image resources.

Tasks:
- [x] Get authorization from the authors of the logo artwork for relicensing it under a libre license (such as CC BY SA)
- [ ] Delete https://github.com/trufont/trufont/blob/master/Lib/defconQt/resources/CC-BY-NC-SA-4.0.txt
- [ ] Edit the toplevel README.md file to include a session describing the overal licensing of the program and of its assets

### PR DESCRIPTION
**Pull request by davelab6**
Fixes #77

Supercedes #87

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# This information can be verified by running:
+# git log --format='%aN <%aE>' | sort | uniq
+
+Adrien Tétar <adri-from-59@hotmail.fr>
+Dave Crossland <dave@lab6.com>
+Denis Jacquerye <moyogo@gmail.com>
+Felipe Corrêa da Silva Sanches <fsanches@metamaquina.com.br>
+Lasse Fister <commander@graphicore.de>
+Nicolas Spalinger <nicolas.spalinger@sil.org>
+Pathum Egodawatta <pathumego@gmail.com>
```
Comment (adrientetar) [inline AUTHORS.txt]: Remove this file.

Comment (davelab6) [inline AUTHORS.txt]: Why?

**Comment by adrientetar** [comment]
Okay I don't have time right now, I will process this another day.

File: AUTHORS.txt (hunk 1 above)
Comment (davelab6) [inline AUTHORS.txt]: I think its reasonable to remove the emails if you wanted to do so; however, I think its unreasonable to not list the copyright authors - I think its essential.

**Comment by davelab6** [comment]
Cool. Also I noticed that you specified LGPL v2.1, not v2.1+ and not 3. I recommend v3+.

**Pull request closed without merging**